fedimint-core/src/encoding/mod.rs

```diff
@@ -787,7 +787,17 @@ where
             fedimint_core::core::ModuleInstanceId::consensus_decode(reader, decoders)?;
         Ok(match decoders.get(module_instance_id) {
             Some(decoder) => {
-                DynRawFallback::Decoded(decoder.decode(reader, module_instance_id, decoders)?)
```

Comment:
This was the mistake. `decoder.decode` does not encode `module_instance_id` and `len` and apparently this path was not covered. But now it is, after a change landed in parallel.